core/domain/services/graphql/connection_resolvers/AbstractConnectionResolver.php

```diff
@@ -35,6 +35,30 @@ protected function getLimit()
         return $limit;
     }
 
+    /**
+     * Get_amount_requested
+     *
+     * This checks the $args to determine the amount requested, and if
+     *
+     * @return int|null
+     * @throws \Exception
+     */
+    // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps
+    public function get_amount_requested()
+    {
+        $amount_requested = parent::get_amount_requested();
+
+        /**
+         * If both first & last are used in the input args, throw an exception as that won't
+         * work properly
+         */
+        if (empty($this->args['first']) && empty($this->args['last']) && $amount_requested === 10) {
+            return 100; // default.
```

Comment:
what is this a default for?

Reply:
number of items to fetch

Reply:
ahh kk thnx...

I thought it was a default price 😆 

> "WUT?!?! $100 default!??!"